### PR TITLE
Update the external packages' commit, and support to update the packages exist.

### PR DESCRIPTION
**Pull request by qiaojbao**
Update the external packages' commit, and support to update the packages exist.

### CODE REVIEW

File: external/fetch_external_sources.py [hunk 1]
```diff
@@ -75,19 +75,17 @@ def CheckOut(self):
         fullDstPath = os.path.join(TargetDir, self.extractDir)
         if not os.path.exists(fullDstPath):
             os.system("git clone " + self.httpsUrl + " " + fullDstPath);
-        else:
-            print("Warning: " + fullDstPath + " exist already, will not download from github again")
-            return;
 
         os.chdir(fullDstPath);
+        os.system("git checkout master");
```
[the remark this answers is not in the document]
Comment (qiaojbao) [reply external/fetch_external_sources.py:80]: You could have a try.

**Comment by amdvlk-admin** [comment]
### Test summary for commit ab12c08ed507902d72dc69e55fbc165e08c845e0
<details><summary>ShaderDB test (Failed: 0/1178)</summary><ul>

<li>Passed: 1176/1178 (99.8%)</li>
<li>Failed: 0/1178 (0.0%)</li>
<li>Not Supported: 2/1178 (0.2%)</li><li>Warnings: 0/1178 (0.0%)</li>
<li>Expected Failed: 0/1178 (0.0%)</li>
</ul></details>

File: external/fetch_external_sources.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (qiaojbao) [reply external/fetch_external_sources.py:80]: Great!
Using 'git fetch' like you suggested better.

**Review by WenqingLiAMD — APPROVED** [review]

**Comment by amdvlk-admin** [comment]
### Test summary for commit 72c4ac5b33c2e2cdbdbfbce36b86ea4f8b51dead
<details><summary>ShaderDB test (Failed: 0/1178)</summary><ul>

<li>Passed: 1176/1178 (99.8%)</li>
<li>Failed: 0/1178 (0.0%)</li>
<li>Not Supported: 2/1178 (0.2%)</li><li>Warnings: 0/1178 (0.0%)</li>
<li>Expected Failed: 0/1178 (0.0%)</li>
</ul></details>

**Pull request merged**